blog/2023-03-16-supabase-authentication/index.md

```diff
@@ -0,0 +1,260 @@
+---
+slug: supabase-authentication-and-rls-protected-tables-on-windmill
+title: Use Supabase Authentication on Windmill to query RLS protected tables
+authors: [adamkov]
+tags: [supabase, authentication, rls, postgresql, windmill, integrate, connect, v2]
+image: ./0-header.png
+---
+
+This example shows how to use Supabase Authentication on Windmill to query tables which have
+[RLS](https://supabase.com/docs/guides/auth/row-level-security) enabled. It can be achieved in two
+ways: using **frontend scripts only** or **involving the backend** as well.
```

Comment:
using scripts executed in the backend and a new experimental method where supabase would be queried by scripts executed in the browser directly.